src/components/BasicInfo.jsx

```diff
@@ -14,11 +14,10 @@ export default function BasicInfo({
     <StyledFieldset>
       <legend>Basic Info</legend>
       <p>Input basic info about yourself.</p>
-      <p className="warning">*Fields are required</p>
       <label htmlFor="firstName">
         First Name{' '}
         <span className="warning" aria-hidden="true">
-          *
+          (required)
```

Comment:
The `aria-hidden` on the `span` hides this from the screen reader. Can you move the text into the label instead?

Reply:
Ahh I see, I missed this change request, apologies, I'll do it now.